### PR TITLE
Fix for scrambled order of columns

### PR DESCRIPTION
**Pull request by PallHaraldsson**
Fixes: #72

### CODE REVIEW

**Comment by PallHaraldsson** [comment]
Hi, @AStupidBear a fix here I think. I copied latter commit from another package, it was slightly different with: DataStructures = "0.17, 0.18"

File: Project.toml [hunk 1]
```diff
@@ -4,6 +4,7 @@ version = "1.4.0"
 
 [deps]
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```
Comment (tkf) [inline Project.toml:7]: As I commented in https://github.com/JuliaLang/julia/issues/37761#issuecomment-699177313, I think we should use OrderedCollections.jl

Comment (PallHaraldsson) [reply Project.toml]: Thanks. FYI: https://github.com/JuliaCollections/OrderedCollections.jl/pull/54

**Comment by PallHaraldsson** [comment]
Just merge this? I'm pretty sure this works. My "but is there a better way?" RFC was about efficiency, while not too important here.

**Comment by tkf** [comment]
I think it's better to wait for @malmaud (although technically I can hit the merge button).

Until then, I think adding a test would help to smooth the review. Maybe test that the order of columns is preserved with the data frames with columns (say) `a, b, c` and `c, b, a`?

Also, since Pandas.jl needs PyCall.jl anyway, I guess we can use PyDict to avoid relying on OrderedCollections.jl here just for one line? Alternatively, it also should be possible to add a column to `pandas.DataFrame` one-by-one.

**Comment by PallHaraldsson** [comment]
If I get my way and Dict will be OrderedDict in Base, then this wouldn't be needed, as of next Julia LTS. But the dependency is light, and may even end up as a stdlib, so I wouldn't worry about that part.

About a test: OrderedDict means the order couldn't be different.

**Comment by malmaud** [comment]
Thanks! I agree with @tkf - this needs a test and I'd rather avoid adding a dependency.

**Comment by PallHaraldsson** [comment]
I did try to avoid this cheap dependency. I probably spent way more time on that (and also the rest) than I have time for, but couldn't see a good way. It's at least very simple this way, and I'm already using it by using my own fork, since not merged here.

**Comment by malmaud** [comment]
Thanks Pall, I just added a test and will merge this now.

**Pull request merged**